docs/table/operations.rst

```diff
@@ -934,3 +937,43 @@ Using multiple columns as ``keys``::
    M31 2012-02-14  16.9  17.3
    M82 2012-02-14  16.2  14.5
    M82 2012-03-26  15.7  16.5
+
+
+.. _set-difference:
+
+Set difference
+--------------
+
+A set difference will tell you the elements that are contained in one set but
+not in the other.  This concept can be applied to rows of a table by using the
+`~astropy.table.setdiff` function. You provide the function with two input
+tables and it will return all unique rows in the first table. If no unique
```

Comment:
This is not really about *unique* rows, since if a table has duplicate rows the `setdiff` function will (should!) still return those duplicates.  (Maybe add a test?)

I would suggest something like the following:
```
A set difference will tell you the elements that are contained in one set but
not in the other.  This concept can be applied to rows of a table by using the
`~astropy.table.setdiff` function.  You provide the function with two input
tables and it returns all rows in the first table which do not occur in the second table.

The optional ``keys`` parameter specifies the names of columns that are used to
match table rows.  This can be a subset of the full list of columns, but both the first
and second tables must contain all columns specified by ``keys``.  If not provided 
then ``keys`` defaults to all column names in the first table.

If no rows are different then the `~astropy.table.setdiff`  function will return an empty table.

The example below illustrates finding the set difference of two observation lists using
a common subset of the columns in two tables.
```